aspects/schema.go

```diff
@@ -323,6 +327,16 @@ func (v *mapSchema) Validate(raw []byte) error {
 	return nil
 }
 
+func validMapKeys(v map[string]json.RawMessage) error {
+	for k := range v {
+		if !validSubkey.Match([]byte(k)) {
+			return fmt.Errorf(`key %q doesn't conform to required format`, k)
```

Comment:
Will the required format be documented somewhere? Might want to include a hint about the format here. Although, it might be hard to consisely explain: "key must be a set of lowercase alphanumeric strings that are seperated by dashes, where at least one character is a letter"?

Reply:
Yeah, I had something about being like a snap name but thought it might be confusing. I grepped for the pattern and saw that the option's validation also just says that it's invalid. I'll make a note for myself as a reminder to say something about this when we add documentation